include/boost/dynamic_bitset/dynamic_bitset.hpp

```diff
@@ -1486,29 +1487,32 @@ dynamic_bitset<Block, Allocator>::find_first() const
     return m_do_find_from(0);
 }
 
-
 template <typename Block, typename Allocator>
 typename dynamic_bitset<Block, Allocator>::size_type
-dynamic_bitset<Block, Allocator>::find_next(size_type pos) const
+dynamic_bitset<Block, Allocator>::find_first(size_type offset) const
```

Comment:
Renaming the variable led to more deltas than are necessary.

Reply:
the intent was "find the first bit from offset" to contrast "find the first bit after position", but in retrospect, "find the first bit from position" is fine. I'll change that back.